python/requirements.in

```diff
@@ -3,12 +3,31 @@ importlib-metadata==4.8
 pipdeptree==2.16.1
 sphinx==7.2.6
 myst-parser==2.0.0
-tox==4.14.1
 sphinx_contributors==0.2.7
 sphinx-copybutton==0.5.2
 furo==2024.1.29
-nose2==0.14.1
+nose2==0.10.0
 coverage==7.4.3
 pyflakes==2.4
 flake8==4.0
+tox == 3.24
+aac == 0.4.3
+# attrs==22.2.0
+# pyrsistent==0.19.3
+# types-PyYAML==6.0.12.2
+# six==1.16.0
+# click==8.1.3
+# pathspec==0.10.3
+# regex == 2022.10.31
+# typing-extensions == 4.8.0
+# Jinja2 == 3.1.2
+# black == 23.9.1
+# MarkupSafe == 2.1.1
+# pygls == 0.13.1
+# fastapi == 0.109.1
+# starlette == 0.35.0
+# anyio < 5, == 3.6.2
+# sniffio == 1.3.0
+# uvicorn == 0.20.0
+# pluggy==1.0.0
```

Comment:
Assume these were commented in case the change over to just install aac caused failure in the pipeline? 

Looks like everything passed, so should be good to fully remove these from the file.

Reply:
Meant to remove those.  Done.